style-rtl.css

```diff
@@ -4807,7 +4807,19 @@ a.to-the-top > * {
 	.pagination-single .next-post .arrow {
 		margin: 0 2rem 0 0;
 	}
+	
+	.pagination-single-inner .next-post .arrow{
+		transform: rotate(-180deg);
+		-ms-transform: rotate(-180deg); /* IE 9 */
+  		-webkit-transform: rotate(-180deg); /* Safari */
+	}
 
+	.pagination-single-inner .previous-post .arrow{
+		transform: rotate(180deg);
+		-ms-transform: rotate(180deg); /* IE 9 */
+  		-webkit-transform: rotate(180deg); /* Safari */
+	}
+	
```

Comment:
@Swapnildhanrale These changes will be overwritten the next time `style-rtl.css` gets rendered. 

This issue must be solved by adding `/*rtl:ignore*/` above the statements in `style.css` which cause this effect. Once you've applied these changes to `style.css` you need to open up a terminal, head over to `/wp-content/theme/twentytwenty` and run `npm install` followed by `npm run build`. 

Let me know if something might be unclear.

Reply:
@nielslange thanks for the info will do the changes and pingback to you...

Reply:
@nielslange I just modified my PR with the latest changes, please check